internal/controller/nodesetcontroller/controller.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2025 Nebius B.V.
```

Comment:
Will we update this file every year at 1 January?)

Reply:
I'd like to have a copyright update machinery connected to the repo